DNN Platform/DotNetNuke.ContentSecurityPolicy/CspParsingExample.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.ContentSecurityPolicy
+{
+    using System;
+
+    /// <summary>
+    /// Example class demonstrating how to parse Content Security Policy headers.
+    /// </summary>
+    public static class CspParsingExample
```

Comment:
Can we move usage examples to the tests project maybe, or maybe tests are good enough examples?